README.md

```diff
@@ -1,10 +1,134 @@
-## Contribution
+# Open A**cademy**: Offering **Online Education for Data Science, Machine Learning, and Deep Learning**

```

Comment:
Ocademy: an open academy for Data Science, Machine Learning, and Deep Learning

Reply:
Thanks for the comment, I will modify it.